buildSrc/build.gradle

```diff
@@ -152,6 +156,18 @@ if (project != rootProject) {
   jarHell.enabled = false
   thirdPartyAudit.enabled = false
 
+  // tests can't  be run with randomized test runner
+  // it's fine as we run them as part of :buildSrc
+  test.enabled = false
+  task integTest(type: Test) {
+    exclude "**/*Tests.class"
+    include "**/*IT.class"
+    testClassesDirs = sourceSets.test.output.classesDirs
```

Comment:
why not project.test.testClassesDirs?

Reply:
I might be wrong on this, but I think the source sets are the record of truth on this. 
Considering that we add a Gradle Test task and not a randomized one, I think it makes more sense than getting it from the randomzied testing task that we don't even use.

Reply:
ok